include/Effect.h

```diff
@@ -125,12 +125,6 @@ class LMMS_EXPORT Effect : public Plugin
 		return 1.0f - m_wetDryModel.value();
 	}
 
-	inline float gate() const
```

Comment:
Per @LostRobotMusic , the knov itself is broken and should not even exist in the first place.

Reply:
I can understand the need to remove the feature if is nonsense.

The question remains though if doing so will break existing songs. Or is the feature so nonsensical that it does not have any effect at all? If it can have an effect then perhaps the loading code should show a warning if a file with one or more active gate settings is encountered so users know that the project is affected by a breaking change.

I don't know the policy of the project but I would expect breaking changes in something like a 2.0 release but not in a 1.3 release.

Reply:
It could technically affect a song that has a long-running effect that outputs very specific low-level samples (below the gate value) and depends on the LMMS autoquit function to cut it off. The consensus we came to in the Discord was that the number of affected projects would be so small, and the effect it would have would be so minor, and the issue with it (notably, that it just cuts off plugins that are making sound and causes audible pops and clicks) significant enough, that we should just remove it. @LostRobotMusic can explain the issues with it more clearly.

Reply:
If it was discussed and the other developers are okay with the removal then I am of course also okay with it. I just wanted to make sure it's okay. :+1:

Reply:
What the Gate knob was _supposed_ to do is disable the effect when the output dipped below a certain volume when the effect was no longer receiving input.  Even if it worked properly, it would be an extremely bad and detrimental feature since it would add discontinuities into the waveform every single time it's enabled or disabled.  It never should have been added in the first place.  With it being broken for the enormous majority of effects, the chances of anybody extensively relying on the feature is near zero.  I'm always extremely strict with maintaining backwards compatibility, but there's no advantage to attempting to maintain it here.  It's a fundamentally broken feature that would still be fundamentally broken even if it worked properly.